### DISCUSSION TITLE
Absolutely need a way to clear cache. I have 253GB of hidden cache files.

### DISCUSSION THREAD

**Discussion by 825i** [iterate-ch/cyberduck#15150] [Q&A]
Guys this ticket #11550 asked for this feature 2 years ago.

I've just learned today that I've lost 253.5GB of storage thanks to cache files from a sync I did 4 weeks ago. This sync was completed almost a month ago and these files still haven't been removed from my system. There is also no way within the app to see the cache I am using nor clean it.

How on earth could you let a fix for an issue like this stay out of the app for so long?

![Screenshot 2023-09-24 204921](https://github.com/iterate-ch/cyberduck/assets/58630825/56c12aa9-b76d-4abb-8003-495dd27c80a1)

and before you accuse this of not being a bug. Why is the cache 5 times the size I set and why have the files remained on my computer 4 times longer than my specified expiry date?

![Screenshot 2023-09-24 205031](https://github.com/iterate-ch/cyberduck/assets/58630825/4a4aad68-048d-456c-b13d-dae2f89b134e)

Especially as it says that my data is totally in sync

![Screenshot 2023-09-24 205149](https://github.com/iterate-ch/cyberduck/assets/58630825/a14768fc-ccf2-4e45-9de5-17c89f2e2526)

**Comment by tatze96** [comment]
The cache size limit applies to each bookmark individually. For example, if you have three bookmarks and use them within the *Smart Synchronization* mode, each bookmark can occupy the selected 50 GiB of cache in size. Files purposely marked to keep offline are excluded from the cache size limit and the cache expiry time.

The cache size can exceed the limit if you upload or open files exceeding the remaining cache space. Mountain Duck will clear out the oldest files from the cache once a day if the cached data exceeds the set limit or the cached item exceeds the set cache expiry time (*Purge files in cache not opened after*). Mountain Duck needs to be running, and the corresponding bookmark must be connected to clear the cache.

You can free up some space manually by right-clicking on a file or folder and choosing `Mountain Duck → Delete on Local Disk` to clear out the file(s) from the cache. You can do the same while right-clicking on white space within the mount to clear out all files from the cache. Note that this will also clear out the files kept offline purposely.

Using the *Delete on Local Disk* option won't prevent Mountain Duck from downloading the files to the cache again.

I generally don't recommend deleting the cache folder, as this can lead to file loss if there are synchronization errors, conflicts, or files that aren't fully synchronized yet.
